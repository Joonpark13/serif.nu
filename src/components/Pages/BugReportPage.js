import React from 'react';

function BugReportPage() {
  return (<div />);
}

export default BugReportPage;
