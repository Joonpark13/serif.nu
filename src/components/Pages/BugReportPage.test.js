import React from 'react';
import { shallow } from 'enzyme';
import BugReportPage from './BugReportPage';

describe('BugReportPage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<BugReportPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
