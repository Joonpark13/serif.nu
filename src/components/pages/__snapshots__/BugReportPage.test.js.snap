// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BugReportPage should render correctly after user acknowledgement 1`] = `
<div
  className="makeStyles-pageContainer-1"
>
  <PageTitle
    title="Report a Bug"
  />
  <div
    className="makeStyles-pageBody-2"
  >
    <iframe
      frameBorder="0"
      height="1100"
      marginHeight="0"
      marginWidth="0"
      src="https://docs.google.com/forms/d/e/1FAIpQLScaXXO7ySVcBk1g0FGdMQqLii0eFaNkh6sOzrTB9KmDe5XmTA/viewform?embedded=true"
      title="Bug report google form"
      width="100%"
    >
      Loading...
    </iframe>
  </div>
</div>
`;

exports[`BugReportPage should render correctly before user acknowledgement 1`] = `
<div
  className="makeStyles-pageContainer-1"
>
  <PageTitle
    title="Report a Bug"
  />
  <div
    className="makeStyles-pageBody-2"
  >
    <div>
      <ForwardRef(WithStyles)
        gutterBottom={true}
        variant="h6"
      >
        I understand that any missing or inaccurate course data is NOT a bug and should not be reported.
      </ForwardRef(WithStyles)>
      <ForwardRef(WithStyles)
        color="primary"
        onClick={[Function]}
        variant="contained"
      >
        Acknowledge
      </ForwardRef(WithStyles)>
    </div>
  </div>
</div>
`;
